### ISSUE TITLE
Simplify navigation for `Running Che locally` documentation

### ISSUE THREAD

**Issue by sunix** [eclipse-che/che#17078]
At the moment, the `Running Che locally` section is hard to follow:
Installing Che in a specific platform is splitted in different part of the doc. It is making the doc hard to follow and understand.
Also some advanced options should be moved to a more advance part of the doc. (minikube LXC partition ... without any mention how to use LXC for installing ...)

![image](https://user-images.githubusercontent.com/650571/83605000-6c024200-a577-11ea-92a0-df17b27a11d7.png)

**Comment by rkratky** [comment]
We have a PR open to sort this out at https://github.com/eclipse/che-docs/pull/1265

**Comment by themr0c** [comment]
@sunix Do you want to keep this issue open?

**Comment by sunix** [comment]
@themr0c OK if you approved that: https://github.com/eclipse/che-docs/pull/1403

**Comment by themr0c** [comment]
OK, done.
